docs/source/preparing_datasets/basic_dataset_conversion.md

```diff
@@ -0,0 +1,313 @@
+# Basic Dataset Conversion
+
+This guide covers how to convert your raw data to MDS format using {class}`streaming.MDSWriter`. Writing to other supported shard formats is very similar. Read more about dataset shard formats in the [Dataset Format](dataset_format.md) guide. For a high-level explanation of how dataset writing works, check out the [main concepts](../getting_started/main_concepts.md#Dataset-conversion) page.
+
+## Configuring dataset writing
+
+Use {class}`streaming.MDSWriter` to convert raw data to MDS format. MDSWriter is like a native file writer; instead of writing the content line by line, MDSWriter writes the data sample by sample. It writes the data into shard files in a sequential manner (for example, `shard.00000.mds`, then `shard.00001.mds`, and so on). Configure {class}`streaming.MDSWriter` according to your requirements with the parameters below:
+
+1. An `out` parameter is an output directory to save shard files. The `out` directory can be specified in three ways:
+ * **Local path**: Shard files are stored locally.
+ * **Remote path**: A local temporary directory is created to cache the shard files, and when shard creation is complete, they are uploaded to the remote location.
+ * **`(local_dir, remote_dir)` tuple**: Shard files are saved in the specified `local_dir` and uploaded to `remote_dir`.
+
+<!--pytest.mark.skip-->
+```python
+out = '/local/data'
+out = 's3://bucket/data' # Will create a temporary local dir
+out = ('/local/data', 'oci://bucket/data')
+```
+
+2. The optional `keep_local` parameter controls if you would like to keep the shard files locally after they have been uploaded to a remote cloud location. To save local disk space, this defaults to `False`.
+
+3. A `column` parameter is a `dict` mapping a feature name or label name with a streaming supported encoding type. `MDSWriter` encodes your data to bytes, and at training time, data gets decoded back automatically to its original form. The `index.json` file stores `column` metadata for decoding. Supported encoding formats are:
+
+| Name   | Class  | Name    | Class   | Name | Class  |
```

Comment:
I think this table is abominable, and should be replaced with a hierarchy which groups the numpy types for example

Reply:
done